orchestra/orchestra_api.py

```diff
@@ -77,9 +77,10 @@ def create_orchestra_project(client,
     return project_id
 
 
-def get_project_information(project_id):
+def get_project_information(project_id, project_ids=[]):
```

Comment:
This leads to some redundant code. Two cleaner approaches to consider:
- `def get_project_information(*project_id)`. Call it with `get_project_information(project_id)` for one id and `get_project_information(*project_ids)` when `project_ids` is a list of ids.
- `def get_project_information(project_ids)` that always expect a list of project_id.

Reply:
i prefer the second FWIW :)